common/src/main/java/com/brandwatch/minibcr/common/repository/MentionRepositoryImpl.java

```diff
@@ -0,0 +1,71 @@
+package com.brandwatch.minibcr.common.repository;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import com.brandwatch.minibcr.common.domain.Mention;
+
+@Repository
+public class MentionRepositoryImpl implements MentionRepository {
+
+    private static final String MENTIONS_TABLE = "mentions";
+    private static final String ALL_FIELDS = " id, text";
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    private final MentionRowMapper mentionRowMapper;
+    private final JdbcTemplate jdbcTemplate;
+    private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
```

Comment:
You don't need to use both of them. ` NamedParameterJdbcTemplate` should do the work.